test/test_all_streams.py

```diff
@@ -75,12 +112,39 @@ def test_run(self):
 
         # Ensure all records have a value for PK(s)
         records = runner.get_records_from_target_output()
+
+        # always assume tags are stale
+        self.tags_are_stale = True
```

Comment:
Is there a reason we want to assume they're stale instead of checking?

Reply:
I'd prefer to check as well but querying via the client or looking at sync'd records only provides tag names and counts with no dates so I assumed it may not be possible.  Will take a closer look at trying to use keyword searches via the client to see if passing in dates works even though the returned records do not have dates.  The web API does expose a date but this is my first (and currently only) guess how to access / use those dates.

Reply:
Confirmed the search endpoint does not support type='tags', fresh out of ideas.  I'll update the comment to provide this context for future code readers.